src/common/filesystembase.h

```diff
@@ -187,24 +187,42 @@ namespace FileSystem {
     QString OCSYNC_EXPORT pathEscape(const QString &s);
 
     namespace SizeLiterals {
-        constexpr unsigned long long operator"" _b(unsigned long long sz)
+        constexpr auto BinaryBase = 1024;
+        constexpr auto DecimalBase = 1000;
+
+        constexpr unsigned long long operator"" _B(unsigned long long sz)
         {
             return sz;
         }
 
-        constexpr unsigned long long operator"" _kb(unsigned long long sz)
+        constexpr unsigned long long operator"" _KiB(unsigned long long sz)
+        {
+            return operator"" _B(sz) * BinaryBase;
+        }
+
+        constexpr unsigned long long operator"" _MiB(unsigned long long sz)
+        {
+            return operator"" _KiB(sz) * BinaryBase;
```

Comment:
`kilo` prefix is usually lowercase (as the only exception in the >1 prefixes), so this should be `kiB`. Same goes for `kB`.

Reply:
True for kB but not for KiB https://en.wikipedia.org/wiki/Byte#Multiple-byte_units